alvr/dashboard/src/dashboard/components/setup_wizard.rs

```diff
@@ -193,3 +236,24 @@ This requires administrator rights!",
         request
     }
 }
+
+#[cfg(target_os = "linux")]
+fn download_and_prepare_audio_script() -> Result<String, Box<dyn Error>> {
```

Comment:
Try to add some spaces in between statements. There is no hard rule for it and it might be a bit arbitrary, but it helps with readability. my two rules are: the return statement should be isolated (especially useful in rust since usually you don't use the return keyword), and group the rest of the code into "blocks" where each block has a easy to explain scope/purpose (although the purpose should be clear from the code itself without comments). aim for no more that 5 statements per block and no more than 5 blocks per function (but i admit this rule is often broken by me).

Reply:
Added a bit of spacing between statements that seemed logical

Reply:
what do you mean by that? i'm a bit unsure

Reply:
add a new line just above `Ok(audio_script_path)`